### DISCUSSION TITLE
any chance at a docker?

### DISCUSSION THREAD

**Discussion by derekcentrico** [alexjustesen/speedtest-tracker#50] [Q&A] [answered]
Is there any chance at a simple docker like what the predecessor offered?

**Comment by derekcentrico** [comment, marked as answer]
I am dumb.  I didn't see updated info and now there is since the first couple alpha builds.  SORRY!  YAY!!!!

**Reply by alexjustesen** [reply]
Glad you figured it out, `v0.1.0-alpha7` which is hot off the press contains major likely breaking changes so make sure to update to that.
